### PR TITLE
Change visibility of hidden cells on PDF export.

### PR DESCRIPTION
**Pull request by hugo-pires**
- [ ] I have read the [Contributor Guide](https://github.com/nteract/nteract/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: packages/desktop/src/notebook/main.css [hunk 1]
```diff
@@ -40,6 +40,21 @@
   .cell.focused .prompt {
     background: var(--pager-bg) !important;
   }
+  .draggable-cell
+  {
+    padding: 0px !important;
+  }
+  .cell-drag-handle
+  {
+    display: none !important;
```
Comment (rgbkrk) [inline packages/desktop/src/notebook/main.css:49]: Smart!

File: packages/desktop/src/notebook/main.css [hunk 2]
```diff
@@ -40,6 +40,21 @@
...
+  }
+  .cell-toolbar-mask {
+    display: none !important;
+  }
+  .invisible
+  {
+    display: none !important;
```
Comment (rgbkrk) [inline packages/desktop/src/notebook/main.css:56]: Excellent.

**Comment by rgbkrk** [comment]
Thanks for going through this!

**Comment by rgbkrk** [comment]
This is looking nice and trim. I just tried it out and love how it looks.

screen shot 2017-10-25 at 9 21 59 am

**Comment by hugo-pires** [comment]
I had a big help from Impactwave team. We will work on a more "compact" theme for printed reports.

Thank you

**Pull request merged**